stories/object-list/object-list.stories.js

```diff
@@ -0,0 +1,1193 @@
+            </span>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Third Attribute
+            </div>
+        </div>
+      </div>
+    </div>
+    </div>
+  </li>
+  <li role="option" tabindex="0" class="fd-list__item fd-object-list__item">
+    <div class="fd-object-list__container">
+      <div class="fd-object-list__intro">
+       Optional inline text</span>
+      </div>
+      <div class="fd-object-list__header">
+        <span class="fd-avatar fd-avatar--s"
+         style="background-image: url('http://lorempixel.com/400/400/nature/4/')">
+        </span>
+        <div class="fd-object-list__header-left">
+         <div class="fd-object-identifier fd-object-list__object-identifier">
+          <p class="fd-object-identifier__title">
+           Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+          </p>
+         </div>
+        </div>
+        <div class="fd-object-list__header-right">
+          <span class="fd-object-number fd-object-number--critical fd-object-list__object-number">
+            <span class="fd-object-number__text">666.00</span>
+            <span class="fd-object-number__unit">Euro</span>
+          </span>
+        </div>
+      </div>
+      <div class="fd-object-list__content">
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              
+              First Attribute
+            </div>
+          </div>
+          <div class="fd-object-list__row-right">
+          <span class="fd-object-status fd-object-status--negative">
+          <i class="fd-object-status__icon sap-icon--status-negative" role="presentation"></i>
+          <span class="fd-object-status__text">Negative</span>
+         </span>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Second Attribute
+            </div>
+          </div>
+          <div class="fd-object-list__row-right">
+            <span class="fd-object-status fd-object-status--critical">
+              <span class="fd-object-status__text">Critical</span>
+            </span>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Third Attribute
+            </div>
+        </div>
+      </div>
+    </div>
+    </div>
+  </li>  
+ </ul>
+</div>
+`;
+
+selection.storyName = 'Selection';
+
+selection.parameters = {
+    docs: {
+        iframeHeight: 305,
+        storyDescription: 'To create row selection list: Create a object list with `fd-list--selection` `fd-list--selection-row` modifiers.'
```

Comment:
Object list items can be selectable by adding the `fd-list--selection` and `fd-list--selection-row` modifier classes to the main element.